crates/sui-core/src/test_utils.rs

```diff
@@ -120,6 +120,12 @@ pub fn create_fake_cert_and_effect_digest<'a>(
 pub fn dummy_transaction_effects(tx: &Transaction) -> TransactionEffects {
     TransactionEffects {
         transaction_digest: *tx.digest(),
+        created: Vec::new(),
+        mutated: Vec::new(),
+        unwrapped: Vec::new(),
+        deleted: Vec::new(),
+        unwrapped_then_deleted: Vec::new(),
+        wrapped: Vec::new(),
```

Comment:
nit: Is this necessary?  These should all be captured by the call to `..Default::default()` below, so we should only add explicit fields here that don't have their default values.

Reply:
Sorry, this was from a bad rebase